File: app/src/main/java/omniex/nl/omniex/di/modules/ActivitiesModule.java
package omniex.nl.omniex.di.modules;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;
import nl.omniex.omniexshopping.ui.app.auth.login.LoginActivity_;
import nl.omniex.omniexshopping.ui.app.auth.register.RegisterActivity_;
import nl.omniex.omniexshopping.ui.app.main.MainMenuActivity_;
import nl.omniex.omniexshopping.ui.app.order.OrderActivity_;
import nl.omniex.omniexshopping.ui.app.profile.address.edit.EditAddressActivity_;
import nl.omniex.omniexshopping.ui.app.splash.SplashActivity_;

@Module
public abstract class ActivitiesModule {

    @ContributesAndroidInjector
    public abstract MainMenuActivity_ contributeMainMenuActivity();

    @ContributesAndroidInjector
    public abstract SplashActivity_ contributeSplashActivity();

    @ContributesAndroidInjector
    public abstract LoginActivity_ contributeLoginActivity();

    @ContributesAndroidInjector
    public abstract RegisterActivity_ contributeRegisterActivity();

    @ContributesAndroidInjector
    public abstract EditAddressActivity_ contributeEditAddressActivity();

    @ContributesAndroidInjector
    public abstract OrderActivity_ contributeOrderActivity();

}

File: app/src/main/java/omniex/nl/omniex/di/modules/ActivitiesModule.kt
package omniex.nl.omniex.di.modules

import dagger.Module
import dagger.android.ContributesAndroidInjector
import omniex.nl.omniex.ui.app.auth.login.LoginActivity_
import omniex.nl.omniex.ui.app.auth.register.RegisterActivity_
import omniex.nl.omniex.ui.app.main.MainMenuActivity_
import omniex.nl.omniex.ui.app.order.OrderActivity_
import omniex.nl.omniex.ui.app.profile.address.edit.EditAddressActivity_
import omniex.nl.omniex.ui.app.splash.SplashActivity_


@Module
abstract class ActivitiesModule {

    @ContributesAndroidInjector
    abstract fun contributeMainMenuActivity(): MainMenuActivity_

    @ContributesAndroidInjector
    abstract fun contributeSplashActivity(): SplashActivity_

    @ContributesAndroidInjector
    abstract fun contributeLoginActivity(): LoginActivity_
    @ContributesAndroidInjector
    abstract fun contributeRegisterActivity(): RegisterActivity_

    @ContributesAndroidInjector
    abstract fun contributeEditAddressActivity(): EditAddressActivity_

    @ContributesAndroidInjector
    abstract fun contributeOrderActivity(): OrderActivity_
}
